src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -813,6 +813,9 @@ def create(notebook_path: str | Path | None = None, **kwargs):
         logger.info(f"Job: {ws.config.host}#job/{job.job_id}")
         return job
 
+    def get_purge_time() -> str:
```

Comment:
Add it as tag also to:
- clusters 
- instance pools
- sql warehouse

Add it a property to:
- schemas
- tables